rest/rule.go

```diff
@@ -341,9 +341,13 @@ func applyNestedInboundRules(
 		s := reflect.ValueOf(value)
 		nestedValues := make([]interface{}, s.Len())
 		for i := 0; i < s.Len(); i++ {
+			val := s.Index(i).Interface()
+			if val == nil {
+				return nil, fmt.Errorf("nested value is nil")
```

Comment:
This can just be `errors.New("nested value is nil"` instead of using `fmt.Errorf` since you're not formatting anything.